src/nodes.coffee

```diff
@@ -1872,9 +1872,17 @@ exports.RegexWithInterpolations = class RegexWithInterpolations extends Base
   constructor: (@call) ->
     super()
 
+  children: ['call']
+
   compileNode: (o) ->
     @call.compileNode o
 
+  astType: -> 'InterpolatedRegExpLiteral'
```

Comment:
This is a non-Babel AST type. Its name vaguely mimics Babel's `RegExpLiteral` type for normal regexes as well as its properties (`flags`, `interpolatedPattern` instead of `pattern`)